examples/handlers/multipart/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "gotham_examples_handlers_multipart"
+description = "An example of decoding requests from an HTML form element"
+version = "0.0.0"
+authors = ["Jacob Budin <self@jacobbudin.com>"]
```

Comment:
Looks like a few of these fields missed being updated when copied from the url encoded example - feel free to add your own name as the author, and let's update the description to specifically say "multipart" to differentiate it from the url encoded form example.

Reply:
@colinbankier You can modify it.

Reply:
@paulzhang - it's easier if you update the last few things - otherwise I have to clone your repo and open a separate PR from my own branch. Committing Github PR suggestions directly doesn't work when I don't have write access to your repo.

Reply:
I added some suggestions - if you could just select "commit suggestion" that'd be great :)